README.md

```diff
@@ -5,7 +5,7 @@ Previous versions (< 3.0.0) implemented the v0 proposal, which was only implemen
 
 ## Usage
 
-Import `CustomElement` and call with your component a tag name __\*__, and a list of attribute names you want to observe:
+Import `register` and call it with your component, a tag name __\*__, and a list of attribute names you want to observe:
```

Comment:
```suggestion
Import `register` and call it with your component, a tag name<strong>*</strong>, and a list of attribute names you want to observe:
```

Sorry, this is unrelated but I think is good to address now; a space before the asterisk makes it look like a wild card (as in, any tag name) when it's really meant to be a qualifier to the tag name.